src/System.Drawing.Common/tests/Helpers.cs

```diff
@@ -17,21 +18,56 @@ public static class Helpers
 
         public static void VerifyBitmap(Bitmap bitmap, Color[][] colors)
         {
-            for (int y = 0; y < colors.Length; y++)
+            try
             {
-                for (int x = 0; x < colors[y].Length; x++)
+                for (int y = 0; y < colors.Length; y++)
                 {
-                    Color expectedColor = Color.FromArgb(colors[y][x].ToArgb());
-                    Color actualColor = bitmap.GetPixel(x, y);
+                    for (int x = 0; x < colors[y].Length; x++)
+                    {
+                        Color expectedColor = Color.FromArgb(colors[y][x].ToArgb());
+                        Color actualColor = bitmap.GetPixel(x, y);
+
+                        if (expectedColor != actualColor)
+                        {
+                            throw new AssertActualExpectedException(expectedColor, actualColor, $"{x},{y}");
+                        }
+                    }
+                }
+            }
+            catch (AssertActualExpectedException ex)
```

Comment:
This is a bit unusual -- can you eliminate the try/catch block, since it looks like we're catching our own exception here?